CHANGELOG.md

```diff
@@ -7,98 +7,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.5.0]
+### Added
+- feat: add support for Terraform `v0.15`
+### Changed
+- feat: upgrade pre-commit hooks to `v0.2.3`
+- build: upgrade build-tools hooks to `v0.11.0`
 ### Fixed
-
 - Preserve case of user names added to teams in plan output
 
 ## [0.4.0]
-
 ### Added
-
```

Comment:
did you change the addon? aligned style was empty line after headers...

Reply:
ok, aligned my vim settings to use prettier for markdown formatting